configs/terraform/environments/prod/artifact-registry.tf

```diff
@@ -41,6 +41,10 @@ resource "google_artifact_registry_repository" "prod_docker_repository" {
       tag_state = "UNTAGGED"
     }
   }
+
+  lifecycle {
+    prevent_destroy = true
+  }
```

Comment:
Can you add support for it in the artifact-registry module as preparation for #12925?